DigitalOffice/Tests/TimeManagementServiceTests/Commands/ChangeWorkTimeCommandTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿ using FluentValidation;
+using LT.DigitalOffice.TimeManagementService.Commands;
+using LT.DigitalOffice.TimeManagementService.Commands.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Mappers;
+using LT.DigitalOffice.TimeManagementService.Mappers.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Validators;
+using Moq;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Commands
+{
+    class ChangeWorkTimeCommandTests
+    {
+        private Mock<IWorkTimeRepository> repositoryMock;
+        private IChangeWorkTimeCommand command;
+        private IValidator<ChangeWorkTimeRequest> validator;
+        private IMapper<ChangeWorkTimeRequest, DbWorkTime> mapper;
+
+        [SetUp]
+        public void SetUp()
+        {
+            repositoryMock = new Mock<IWorkTimeRepository>();
+            mapper = new WorkTimeMapper();
+            validator = new ChangeWorkTimeRequestValidator();
+
+            command = new ChangeWorkTimeCommand(validator, repositoryMock.Object, mapper);
+        }
+
+        [Test]
+        public void ShouldThrowsExceptionWhenDataIsInvalid()
+        {
+            var request = new ChangeWorkTimeRequest
+            {
+                Id = Guid.NewGuid(),
+                WorkerUserId = Guid.NewGuid(),
+                StartTime = new DateTime(2020, 1, 1, 1, 1, 1),
+                EndTime = new DateTime(2020, 2, 2, 2, 2, 2),
+                Title = "E",
+                ProjectId = Guid.NewGuid(),
+                Description = "ExampleDescription"   
+            };
+
+            Assert.Throws<ValidationException>(() => command.Execute(request));
```

Comment:
Where is verifying that method of repositoryMock was not called?

Reply:
How can I check it?